.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```

Comment:
Suggest that we segment features for **code** and the **data standard** because they should likely be handled differently.

Reply:
I agree, there's basically no code yet, so I treated these as data standard templates, I could make that a little more explicit and add an issue template for infrastructure code.

Reply:
There's a "spec feature" label and a "bug" label in the issue tracker, but nothing specific to "spec bug", should we add that, or is there some other way of labeling issues to make clear they apply to the spec? Could either combine atomic labels, e.g., "spec" + "bug", "code" + "bug"; or create combined labels, "spec bug", "spec feature". I might have a slight preference for the atomic labels, but will happily defer to the rest of the team.